Gordon360/Services/RecIM/ActivityService.cs

```diff
@@ -66,7 +67,7 @@ public IEnumerable<ActivityExtendedViewModel> GetActivities()
                                 SoloRegistration = a.SoloRegistration,
                                 Logo = a.Logo,
                                 Completed = a.Completed,
-                                Series = a.Series
+                                Series = a.Series.Where(s => s.StatusID !=0)
```

Comment:
```suggestion
                                Series = a.Series.Where(s => s.StatusID != 0)
```
Just a small syntax issue